### ISSUE TITLE
Ulnar styloid large 

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10834]
**Preferred term label:**
Ulnar styloid large 

**Synonyms**

**Definition (free text, please give PubMed ID)**
AP x-ray of the wrist measuring the ulnar styloid length >6mm or the ulnar styloid process index (USPI) >0.21. USPI is calculated by the difference between the ulnar styloid length and the ulnar variance, then divided by the transverse diameter of the ulnar head.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Ulnar styloid impaction syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6001117
- label: Hypertrophic ulnar styloid process
- parents: [Abnormal morphology of ulna (HP:0040071)](https://hpo.jax.org/app/browse/term/HP:0040071)
- pmid: PMID:31192047
- def: Excessively long styloid process of the ulna.
- comment: Ulnar styloid impaction syndrome involves impaction between the tip of the ulnar styloid process and the triquetrum bone. This syndrome is most commonly seen in patients with excessively long or hypertrophic ulnar styloid processes and is not necessarily associated with positive ulnar variance. The ulnar styloid process index (USPI), calculated as the difference between the ulnar styloid length and the ulnar variance divided by the transverse diameter of the ulnar head, is commonly used to assess the size of the ulnar styloid. An excessively long ulnar styloid has a length greater than 6 mm or a USPI more than 0.21. Repetitive friction of the proximal pole of the triquetrum and the ulnar styloid tip leads to chondromalacia, synovitis, and ulnar-sided wrist pain.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10834
- synonyms: Ulnar styloid large(NONE)
